### ISSUE TITLE
Fatal JS error using media_gallery with small images

### ISSUE THREAD

**Issue by carlaiau** [htmlburger/carbon-fields#796]
## Version

  - Carbon Fields: 3.1.2
  - WordPress: 5.3
  - PHP: 7.3

## Expected Behavior
Media Gallery, adding any image should be allowed, irrespective of it's size.

## Actual Behavior
Media Gallery, when adding a small image, such as a 64x64 sized image we experience a fatal JavaScript bug that is echoed into the console. 

```
TypeError: "a.sizes.thumbnail is undefined"
```

## Container definition
```php
Container::make('theme_options', 'Hero/Grid backup Images')
  ->add_fields([
    Field::make('media_gallery', 'page_backup_images', 'Page')
  ])
```

## Steps to Reproduce the Problem

  1. Create a small image, by screenshot or something similar, as long as the dimensions are below the site's thumbnail specification. 
  2. Carbon Fields then looks for this thumbnail image, which I presume is not actually getting generated dynamically

## Comments
Looks related to https://github.com/htmlburger/carbon-fields/issues/677

**Comment by lstoyanoff** [comment]
Hi, there was a similar issue https://github.com/htmlburger/carbon-fields/issues/734 which I believe was fixed in 3.1.3.

**Comment by carlaiau** [comment]
I can confirm this issue has been resolved! Thanks
